crates/binstalk/src/fetchers/gh_crate_meta.rs

```diff
@@ -290,6 +310,8 @@ impl leon::Values for Context<'_> {
 
             "binary-ext" => Some(Cow::Borrowed(self.binary_ext)),
 
+            "subcrate_prefix" => Some(Cow::Borrowed(self.subcrate_prefix.unwrap_or(""))),
```

Comment:
hmm all the other ones are kebab, not snake, let's keep that convention.

also the struct field above needs a doc comment

Reply:
also I think this should just be called subcrate, there's no inherent requirement that it's a prefix I think?, we just auto-discover it that way

Reply:
> also I think this should just be called subcrate, there's no inherent requirement that it's a prefix I think?, we just auto-discover it that way

I called it prefix because it also contains the '%2F' (escaped '/').
TBH, I'm not good at naming, but I also don't think subcrate is a good name due to the additional slash.

Maybe something like subcrate-url-part is a good name?

Reply:
Hmm, I'd missed that detail. I think we should:
- use `subcrate`, not have it include the `/` (because it could be `foo-v1.2.3` in the version tag/URL instead)
- add extra release paths with `subcrate` instead of replacing the existing (see below example)
- use leon's "has_key" capability to only render/use those templates that do have `subcrate` when subcrate isn't `None`.

so for github we'd have

```
{ repo }/releases/download/{ version }
{ repo }/releases/download/v{ version }
{ repo }/releases/download/{ subcrate }/{ version }
{ repo }/releases/download/{ subcrate }/v{ version }
{ repo }/releases/download/{ subcrate }-{ version }
{ repo }/releases/download/{ subcrate }-v{ version }
```

and the bottom four would only get used if `subcrate` is available.